csrc/mmdeploy/preprocess/elena/elena_int.h

```diff
@@ -0,0 +1,40 @@
+#include <math.h>
```

Comment:
The definition in this file looks coming from 'third_party/CVFusion/include/CodeGen/DeviceCodegen.h'
I don't think mmdeploy has the obligation to maintain `elena_int.h`. If `DeviceCodegen.h` has errors, please pull a request to their repo